end_to_end/nlp_mlops_company_sentiment/03_nlp_company_earnings_analysis_lambda_async.ipynb

```diff
@@ -0,0 +1,1343 @@
+{
```

Comment:
<p>Recent financial-themed JumpStart notebooks add a legal disclaimer:</p><blockquote><strong>Legal Disclaimer</strong>: This example notebook is for demonstrative purposes only. It is not financial advice and should not be relied on as financial or investment advice.</blockquote><p>I think it would make sense to add the same for any notebooks analyzing financial information.</p><p>See<a href=" https://github.com/aws/amazon-sagemaker-examples/blob/master/sagemaker-jumpstart/multimodal_tabtext/PPP_tabtext_ML.ipynb" target="_blank"> https://github.com/aws/amazon-sagemaker-examples/blob/master/sagemaker-jumpstart/multimodal_tabtext/PPP_tabtext_ML.ipynb</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3056/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
agreed